src/components/program/ProgramContentMenu.tsx

```diff
@@ -450,7 +483,9 @@ const SortBySectionItem: React.VFC<{
   const [exerciseId] = useQueryParam('exerciseId', StringParam)
   const programContentProgress = useProgramContentProgress()
   const { specifyExamExercise } = useSpecifyExamExercise(exerciseId || '')
-  const { data: materials, loading: materailLoading } = useProgramContentMaterial(programContent.id)
+  // const { isEquityProgramContent, loadingProgramContent } = useProgramContentById(programId, programContent.id)
```

Comment:
delete unused code here

Reply:
I missed deleting this, thank you for checking it.